packages/main/src/ComboBox.ts

```diff
@@ -463,8 +463,8 @@ class ComboBox extends UI5Element {
 		this._selectMatchingItem();
 		this._initialRendering = false;
 
-		const slottedIconsCount: number = this.icon.length || 0;
```

Comment:
why removing, it's not a bad practice

Reply:
I noticed that number is already set by the expression and might be redundant in this case.